grafana_objects/client_grafana_objects_get_by_uid.go

```diff
@@ -0,0 +1,50 @@
+package grafana_objects
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	logzio_client "github.com/logzio/logzio_terraform_client"
+	"github.com/logzio/logzio_terraform_client/client"
+)
+
+const (
+	grafanaObjectsDashboardsByUID = grafanaObjectServiceEndpoint + "/uid/%s"
+)
+
+// Get allows getting a Grafana objects configuration.
+// https://docs.logz.io/api/#operation/getDashboarById
+func (c *GrafanaObjectsClient) Get(objectUid string) (*GetResults, error) {
```

Comment:
Whenever you initiate a call to the Logzio api, please use the CallLogzioApi function instead.
You can see an example here:
https://github.com/logzio/logzio_terraform_client/blob/master/archive_logs/client_archive_logs_retrieve.go#L19

Reply:
done